README.md

```diff
@@ -250,6 +250,14 @@ Set the priority per instance using the `TaskInstance.Builder`:
             .scheduledTo(Instant.now()));
 ```
 
+You can also set the default priority for all tasks of a given type:
+
+```java
+Tasks.recurring("my-task", FixedDelay.ofSeconds(5))
+    .setDefaultPriority(Priority.LOW)
```

Comment:
Maybe skip set, i.e. `.defaultPriority(Priority.LOW)`, to harmonize with the other methods..

Reply:
Problem was that we use `defaultPriority` for the name of the field, so the method can't have the same name.

Can rename to `_defaultPriority`, but no other private vars have underscores 😅

Reply:
Will fix it by using `priority` for the private field.

Reply:
ah nvm, seems its ok to have same name of private field and method